allensdk/brain_observatory/behavior/session_apis/abcs/behavior_ophys_base.py

```diff
@@ -4,22 +4,28 @@
 import numpy as np
 import pandas as pd
 
-from allensdk.brain_observatory.behavior.internal.behavior_base import BehaviorBase
+from allensdk.brain_observatory.behavior.session_apis.abcs import BehaviorBase
 from allensdk.brain_observatory.behavior.image_api import Image
 
 
 class BehaviorOphysBase(BehaviorBase):
     """Abstract base class implementing required methods for interacting with
-    behavior+ophys session data.
+    behavior + ophys session data.
 
     Child classes should be instantiated with a fetch API that implements these
     methods.
     """
 
+    @abc.abstractmethod
+    def get_ophys_experiment_id(self) -> Optional[int]:
```

Comment:
The relationship is one-to-one for scientifica sessions and experiments. But you can have many experiments per session for mesoscope experiments. This will need to be changed when it comes time to include mesoscope functionality but I considered it a bit out of scope for what this PR should be addressing.